### ISSUE TITLE
Some guns cease working after weapon switch.

### ISSUE THREAD

**Issue by FleshJeb** [SpannerMonkey/SM_Armory#6]
Situation:
Craft with APC MG2, Spanner's Phalanx CIWS, and a PAIR of Bofors 57mm.
-Can fire the 20mms with no problem and switch back and forth between them
-Switch to 57mm, fire or don't fire.
-Switch back to 20mms, and they won't fire, or will only fire one round.

Tried testing with the BDA Vulcan turret, and M230 30mm chaingun. Works fine for switching.
The issue is using something with non-standard ammo. The 57mm breaks them. Switching to something 30mm, and switching back to the MG2 and the CIWS works.

Of note, the 57mm breaks the BDA Vulcan as well. Only switching to the M230 fixes it.

Did a bunch more testing. The issue is with having a PAIR of the 57mm turret.
The same problems arise with a pair of M230s, MAC30s, 75mms, and 90mms.

**In summary: A pair (or more?) of weapons breaks 20mms.
Switching to a SINGLE M230 30mm fixes it until you fire the offending weapon again.
I think this a general BDArmory Continued problem**

Possible conflicts: Using the modpack for Tape and Beardy's Kerbol Rising series.

**Comment by FleshJeb** [comment]
More testing indicates that it's probably a general BDA problem, and possibly specifically related to changes made in the specific modpack.

Better rundown here: https://github.com/PapaJoesSoup/BDArmory/issues/463

**Comment by SpannerMonkey** [comment]
Hi closing. As the problem is mod interaction in an unsupported modpack.
Older versions ie pre damage overhaul versions are not supported

Note for the future, use of my parts in random mod packs is not and will never supported.
